listenbrainz/webserver/views/api_compat.py

```diff
@@ -2,6 +2,7 @@
 import time
 import json
 import re
+import listenbrainz.db.user as db_user
```

Comment:
In general, if you find unalphabetized imports, you should alphabetize them. Fine for now though.